datafusion/sqllogictest/test_files/subquery.slt

```diff
@@ -878,13 +878,8 @@ Projection: t1.t1_id, CASE WHEN __scalar_sq_1.__always_true IS NULL THEN Int64(2
 --------Aggregate: groupBy=[[t2.t2_int, Boolean(true) AS __always_true]], aggr=[[COUNT(UInt8(1)) AS COUNT(*)]]
 ----------TableScan: t2 projection=[t2_int]
 
-query II rowsort
+query error DataFusion error: Arrow error: Invalid argument error: arguments need to have the same data type
```

Comment:
I'm tracking this one down now

Reply:
This bug was fixed by https://github.com/apache/arrow-datafusion/pull/8758/commits/1e4eca227c34a5130e97b4ab07369f71c0525311, right?

Reply:
Yes